### ISSUE TITLE
Trained pytorch checkpoint

### ISSUE THREAD

**Issue by midskavid** [noahcao/Pixel2Mesh#4]
I loaded the file vgg16-p2m.pth provided in pretrained folder and ran the entrypoint_predict script on a folder of cars (provided in data_tf/02958343/ ). However the output mesh is too bad! looking at the results, it seems that the model that I have loaded is not trained.

Could you please share a pytorch pretrained p2m model? Or am I missing something?

Thanks!

**Comment by ultmaster** [comment]
Hi. We believe that pretraining the vgg16 backbone on ImageNet will accelerate and stablize the training, but unfortunately such pretraining seems to backfire and will blow the loss.

For the actual pretrained checkpoints (for both VGG and ResNet), please refer to the "pretrained checkpoint" section in README.

**Comment by midskavid** [comment]
I get this error :
size mismatch for gcns.0.conv1.loop_weight: copying a param with shape torch.Size([3843, 192]) from checkpoint, the shape in current model is torch.Size([963, 192])

when loading resnet.pth.tar and running entrypoint_predict.py

What am I missing?

**Comment by ultmaster** [comment]
Sorry for slow reply. I think I've missed the email.

Are you sure you are referring to the right yml in `--options`?

**Comment by ultmaster** [comment]
Any updates?

**Comment by zshyang** [comment]
Hi,
Do you know how to download the dataset and unzip it from the link below?
https://drive.google.com/open?id=131dH36qXCabym1JjSmEpSQZg4dmZVQid

**Comment by zshyang** [comment]
Hi,
Do you know how to use their provided YAML files and pretrained weights downloaded from here to do the evaluation?
To my understanding, I should use the downloaded meta folder. Then use the provide pretrained weights.
The command I use is 
```
python entrypoint_eval.py --name xxx --options experiments/default/tensorflow.yml --checkpoint datasets/data/pretrained/vgg16-p2m.pth 
```
But it seems that it only generated some meaningless meshes. And the score is really low.
I am not sure what I did wrong.
![Screenshot from 2020-09-22 02-08-41](https://user-images.githubusercontent.com/57812785/93863819-de0eac00-fc78-11ea-91b5-0235f5b4487b.png)
